### PR TITLE
Handle edge case to correctly count total bit size and calculate byte array

### PR DESCRIPTION
**Pull request by jaikit**

### CODE REVIEW

File: src/test/java/com/iab/gdpr/VendorConsentTest.java [hunk 1]
```diff
@@ -133,4 +177,30 @@ public void testRangeEntryConsent() {
         assertFalse(consent.isVendorAllowed(3));
         assertTrue(consent.isVendorAllowed(27));
     }
+    
+    @Test
+    public void testLongRangeEntry() {
```
Comment (nabcos) [inline src/test/java/com/iab/gdpr/VendorConsentTest.java]: This PR seems to have duplicated this test method, 218a54cfabdf704182cea31d4483b1eba43e05f4 does not build anymore for me.

 @lanusau Did you run the `gradlew test` before merging? Can you add travis integration for this repository? There's already a `.travis.yml` executing the tests by default, so that should be straightforward?

Comment (lanusau) [reply src/test/java/com/iab/gdpr/VendorConsentTest.java]: My bad. I will fix this. Yes, we need to add travis integration

Comment (lanusau) [reply src/test/java/com/iab/gdpr/VendorConsentTest.java]: This was fixed in #23, I merged that and it should be passing now.

Comment (nabcos) [reply src/test/java/com/iab/gdpr/VendorConsentTest.java]: Alright, thanks!
